pkg/security/mcs/label.go

```diff
@@ -0,0 +1,178 @@
+package mcs
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+const maxCategories = 1024
+
+type Labels struct {
```

Comment:
This leads me to believe that we should update `SecurityContext` upstream (post 3.0) to use better terminology too.  The fields were largely based on how selinux items were passed via the docker run flags which uses "level" but encompasses level, category, and range.

reference: https://docs.docker.com/reference/run/#security-configuration